File: examples/go/about.go
package main

const (
	AppVersion = "0.1.0"
	DonateURL  = "https://github.com/sponsors"
)

// AboutSummary is the CLI About slice (version + donate, no crash payload).
func AboutSummary() string {
	return "golden-path " + AppVersion + " donate " + DonateURL
}

File: examples/rust/src/about.rs
//! CLI About slice: version + donate URL (no crash payload).

pub const APP_VERSION: &str = "0.1.0";
pub const DONATE_URL: &str = "https://github.com/sponsors";

pub fn summary() -> String {
    format!("golden-path {APP_VERSION} donate {DONATE_URL}")
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn includes_version_and_donate() {
        let text = summary();
        assert!(text.contains(APP_VERSION));
        assert!(text.contains("donate"));
    }
}
